Sources/OktaDirectAuth/Internal/Authentication Factors/SecondaryFactor.swift

```diff
@@ -63,6 +84,8 @@ extension DirectAuthenticationFlow.SecondaryFactor: AuthenticationFactor {
             return .otpMFA
         case .oob:
             return .oobMFA
+        case .webAuthn, .webAuthnAssertion(_):
```

Comment:
will webauthn in MFA use case be added separately? if so this grant type will probably change to `urn:okta:params:oauth:grant-type:mfa-webauthn` and that will create a problem with resuming the flow using secondary factor similar to the issue https://github.com/okta/okta-mobile-swift/issues/175.

Reply:
No, according to the spec, the grant-type is the same for both primary auth vs MFA auth.  However, you did help me point out one gap, where the list of `challenge_types_supported` will return `urn:okta:params:oauth:grant-type:mfa-webauthn` when initiating a WebAuthn challenge as an MFA factor. So I need to make sure that's supported in this code.